src/Microsoft.OData.Edm/ExtensionMethods/AnnotationsTargetHelper.cs

```diff
@@ -0,0 +1,205 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="AnnotationsTargetHelper.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+namespace Microsoft.OData.Edm
+{
+    using System.Collections.Generic;
+    using Microsoft.OData.Edm.Vocabularies;
+
+    /// <summary>
+    /// AnnotationsTarget helper
+    /// </summary>
+    internal static class AnnotationsTargetHelper

```

Comment:
There are at least 12 execution paths that do not have test coverage in this file currently.